spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe '/users/:user_id/posts', type: :request do
+  let(:first_user) { User.create(name: 'John', photo: 'https://example.com/john.jpg', bio: 'Some bio') }
+
+  describe 'GET /index' do
+    it 'renders a successful response' do
+      get "/users/#{first_user.id}/posts"
+
+      expect(response).to be_successful
+    end
+  end
+
+  describe 'GET /index' do
+    it 'renders an h2 text' do
+      get "/users/#{first_user.id}/posts"
+
+      expect(response.body).to include('<h2>List of all posts for a given user</h2>')
+    end
+  end
+
```

Comment:
* Good job writing tests for each controller but as mentioned in the [requirements ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=Make%20sure%20that%20for,includes%20correct%20placeholder%20text)
```
Make sure that for each action you check:
If response status was correct.
If a correct template was rendered.
If the response body includes correct placeholder text
```
* Kindly write all these tests for each action for example post controller(index action has 3 tests, while show action has 3 tests too) all tests should cover the required specifications

Reply:
thank you, dear @Davidosky007, I have created 3 tests for `index` and 3 tests for `show`, for both, `users` and `posts` so now there is 12 tests in total 
![image](https://github.com/danifromecuador/rails_blog/assets/116839451/59451122-a6b0-4973-9c34-360142e43704)